File: PRAK402-2410817310009-Muhammad_Ghazi_Rakhmadi.c
#include <stdio.h>

int main() {
    int input;
    printf("Input: ");
    scanf("%d", &input);

    printf("");
    for (int i = 1; i <= input; i += 2) {
        printf("%d ", i);
    }

    printf("\n");
    for (int i = input; i >= 2; i--) {
        if (i % 2 == 0) {
            printf("%d ", i);
        }
    }
}

File: PRAK402-2410817310009-Muhammad_Ghazi_Rakhmadi.py
n = int(input("Input: "))

print("", end=" ")
for i in range(1, n + 1):
    if i % 2 != 0:
        print(i, end=" ")
print() 

print("", end=" ")
for i in range(n, 0, -1):
    if i % 2 == 0:
        print(i, end=" ")
print()